apps/verification-portal/src/lib/components/NFTImage.svelte

```diff
@@ -77,7 +80,7 @@
     {#if isMounted && !isError}
       <!-- Render the image when it's loaded -->
       <img
-        loading="eager"
+        loading="lazy"
```

Comment:
Might need to add this as an additional prop, eager is necessary for images in the Header NFT to have priority loading and improve the first contentful paint

Reply:
I added this as param in my next client's refactoring PR. Thx.